exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,126 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
+  ],
+  "cases": [
+    {
+      "description": "partial garden tests",
+      "cases": [
+        {
+          "description": "test Alice's garden",
```

Comment:
I don't know if it's just me, but it seemed more important to me to say that this tests a garden with only a single student, rather than stating the name of the student (already inferrable from `"student":` key)